plugin/arcconf/arcconf.py

```diff
@@ -14,7 +14,6 @@
 # License along with this library; If not, see <http://www.gnu.org/licenses/>.
 #
 # Author: Gris Ge <fge@redhat.com>
-#         Joe Handzik <joseph.t.handzik@hpe.com>
```

Comment:
Agreed, just leave the author list the way it was, and append your name and contact information to it, @brraghav